content/blog/2023-03-14-mining-ethereum-classic-with-an-ipollo-asic-through-2miners-pool/index.md

```diff
@@ -0,0 +1,178 @@
+
+![iPollo connection.](./3.png)
+
+## 4. Get the iPollo ASIC Mining Rig URL in Your Local LAN
+
+Now that your mining rig is connected and powered, you have to find its IP address in your local LAN so you may access its app from your browser from your computer (your computer needs to be connected through ethernet cable or WiFi to the local LAN as well).
+
+To do this, go to the IP address of your LAN modem (this may be Googled or requested to your internet provider if you don’t have it) access the app of your local LAN modem, find the iPollo machine in the “connected devices” section or equivalent, and then click on it to see its specific IP address.
+
+In our example we use Cox as our internet provider, and the iPollo rig IP address is 192.168.0.180.
+
+![iPollo URL.](./4.png)
+
+## 5. Log In to Your iPollo ASIC App
+
+Once you know the IP of your iPollo mining machine you can type it on your browser to access its app, which is hosted in the machine itself.
+
+In our example, we typed 192.168.0.180 on our browser (do not enter the prefixes “https” no “http” as this is a local connection) and we accessed the login page of the iPollo app.
+
+To log in, you must type the Username “root” and the Password “root” (don’t worry, in the next step you will change the password).
+
+![Log in page.](./5.png)
+
+## 6. Change the Password on Your iPollo ASIC Miner App
+
+In this step you need to change the password of the app on your iPollo for security reasons.
+
+For this, go to the “Password Change” tab on the left menu, then enter the new password and press “Save”.
+
+![Change password.](./6.png)
+
+## 7. Set Up Your Miner Credentials
+
+To set up your miner credentials (the configuration data so your iPollo miner will be pointing to your account on the 2miners pool) go to the “Miner Configuration” tab on the left menu.
+
+Once on that screen, change the default mining algorithm from ETHASH to ETCHASH on the dropdown menu at the center top of the page.
+
+![Miner configuration.](./7.png)
+
+## 8. Get Miner Settings Information From The 2miners Mining Pool
+
+Now you need to go to 2miners to get your credentials or configuration data. For this, go to:
+
+https://etc.2miners.com/help 
+
+Once there, create an ETC wallet as per the instructions or use an existing wallet you already have, if you wish.
+
+Then, get the instruction shown further below in that page for “ASIC Settings”. The instructions are:
+
+**URL: stratum+tcp://etc.2miners.com:1010**
+
+**Worker: YOUR_WALLET_ADDRESS.ASIC_ID**
+
+**Password: x**
+
+![Miner settings from 2miners.](./8.png)
+
+## 9. Enter Your Configuration on Your iPollo
+
+Now, you have to go back to the iPollo ASIC miner app and enter your configuration in the “Miner Configuration” page. In our example, we used the account shown below with the extension “ASIC_ID”, but you may use any ID number like ASIC_1 or ASIC_28, etc. The ID is in case you have several miners pointing to 2miners so you may identify them independently
```

Comment:
Line 130: Usage; Add punctuation
```suggestion
Now, you have to go back to the iPollo ASIC miner app and enter your configuration on the “Miner Configuration” page. In our example, we used the account shown below with the extension “ASIC_ID”, but you may use any ID number like ASIC_1 or ASIC_28, etc. The ID is in case you have several miners pointing to 2miners so you may identify them independently.
```